app/javascript/components/rules/index.js

```diff
@@ -1,67 +1,102 @@
 import React from "react"
 import { AppProvider, Layout, Card, Badge } from '@shopify/polaris'
 
-class Index extends React.Component {
+class SingleRule extends React.Component {
   render () {
+    let rule = this.props.rule
```

Comment:
My linter complains about the lack of ;, but I don't think we have settled on any formal rule yet.